openedx/core/djangoapps/schedules/templates/schedules/edx_ace/common/course_cta.html

```diff
@@ -0,0 +1,27 @@
+{% load i18n %}
```

Comment:
nit: "course CTA" is a little vague? maybe "return_to_course_cta.html"?

Reply:
sounds good.  renamed.